pdl-compiler/Cargo.toml

```diff
@@ -34,6 +36,8 @@ serde_json = "1.0.94"
 argh = "0.1.12"
 syn = "2.0.58"
 prettyplease = "0.2.17"
+once_cell = "1.21.3"
```

Comment:
I think I might prefer the following, for as long as `java` is the only feature relying on it.

```
[dependencies.java]
genco = "0.17.10"
```